### ISSUE TITLE
Profile photo error + visited location

### ISSUE THREAD

**Issue by zarevskaya** [seanmorley15/AdventureLog#110]
Hello again, although I may as you know, when I try to send a profile picture, it doesn't work (yet):

![Screenshot_20240710_173619](https://github.com/seanmorley15/AdventureLog/assets/60230221/7f608de7-8014-474e-9343-dba0ca8b5c5f)
![Screenshot_20240710_173807](https://github.com/seanmorley15/AdventureLog/assets/60230221/d9bcebde-dcc3-42d0-bef4-b63ab9527f15)

Same for the place visited:

![Screenshot_20240710_174533](https://github.com/seanmorley15/AdventureLog/assets/60230221/3223b19f-e639-48b5-ae60-c7f21b9e567d)

**Comment by seanmorley15** [comment]
The adventure images and profile pictures are served to the frontend through the nginx container. When a request is made to /media/ to the backend, nginx takes the request and serves the correct image. This might not be the cause of the problem though. The variable for the backend container `PUBLIC_URL` is what actually needs to be linked to the nginx because this will serve as the url for media files. So if your nginx was registered to cdn.yourdomain.com the `PUBLIC_URL` would be exactly that. Ex: this is what I used for the hosted version `PUBLIC_URL='https://cdn.adventurelog.app'`

Sorry for the confusing variable names! I just started making a documentation site (https://docs.adventurelog.app/) and am working on installation instructions to hopefully clear up confusing setup issues.

Thanks a lot!
